tests/enospc-refresh-30mins.yaml

```diff
@@ -6,8 +6,8 @@ tests: !mux
     enospc:
```

Comment:
@amoskong, do we still use this file?
I think it's an old one and can be removed.

Reply:
Yes, we can remove it.